Tella/Scenes/Uwazi/ViewModel/AddTemplateVM.swift

```diff
@@ -68,8 +71,7 @@ class AddTemplateViewModel: ObservableObject {
     }
     
     func downloadTemplate(template: CollectedTemplate) {
-        var template = template
-        self.downloadTemplate(template: &template)
+        self.downloadTemplateWithRelationships(template: template)
         self.templateItemsViewModel.first(where: {template.templateId == $0.id})?.isDownloaded = true
```

Comment:
maybe we in need to include `self.templateItemsViewModel.first(where: {template.templateId == $0.id})?.isDownloaded = true` inside `fetchRelationshipEntities` result 



    func downloadTemplate(template: CollectedTemplate) {
        self.isLoading = true
        entityFetcher?.fetchRelationshipEntities(template: template) { relationships in
            template.relationships = relationships
            self.saveTemplate(template: template)
            self.isLoading = false
            self.templateItemsViewModel.first(where: {template.templateId == $0.id})?.isDownloaded = true
        }
    }